src/client/Microsoft.Identity.Client/Platforms/Features/RuntimeBroker/WamAdapters.cs

```diff
@@ -206,6 +211,19 @@ private enum ResponseStatus
             return authParams;
         }
 
+        private static void DecorateExceptionWithRuntimeErrorProperties(MsalException exception, Error runtimeError)
+        {            
+            var result = new Dictionary<string, string>()
+            {
+                { "RuntimeContext", runtimeError.Context },
```

Comment:
What's the guideline for property bag key names? Should these be in some constant class? I assume changing the names of these afterwards will also be a breaking change?

Reply:
Moved to MsalException as constants.